torch/testing/_internal/inductor_utils.py

```diff
@@ -41,18 +42,13 @@ def test_cpu():
 
 HAS_GPU = HAS_CUDA or HAS_XPU
 
-GPUS = ["cuda", "xpu"]
+GPU_TYPE = get_gpu_type()
```

Comment:
Let's refactor this so it doesn't initialize the GPU subsystem on import.

Reply:
Hi @jansel, HAS_CUDA/HAS_XPU/HAS_GPU are already in the header of this file, and in Inductor UT it is imported to get HAS_CUDA/HAS_XPU/HAS_GPU. Seems onece import `torch.testing._internal.inductor_utils`, the GPU subsystem is always initialized.